src/main/java/jenkins/branch/CustomOrganizationFolderDescriptor.java

```diff
@@ -59,17 +59,50 @@
         super(TopLevelItem.class); // do not register as OrganizationFolder
         this.delegate = delegate;
     }
-    
+
     @Override
     public String getId() {
         return OrganizationFolder.class.getName() + "." + delegate.getId(); // must be distinct from OrganizationFolder.DescriptorImpl
     }
-    
+
     @Override
     public String getDisplayName() {
         return delegate.getDisplayName();
     }
 
+    /**
+     * Needed if it wants SCMNavigator implementations are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string with the Item description.
+     */
+    public String getDescription() {
+        return delegate.getDescription();
+    }
+
+    /**
+     * Needed if it wants SCMNavigator implementations are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string it represents a ItemCategory identifier.
+     */
+    public String getCategoryId() {
+        return "nested-projects";
```

Comment:
`CustomOrganizationFolderDescriptor` is used for `SCMNavigator` implementations and we have decided that this kind of Item would be **nested-projects**.

Other option is [this|https://github.com/jenkinsci/scm-api-plugin/pull/9#discussion_r58925376].

IMO both are right.